src/components/CVE-2022-36804/CVE-2022-36804.tsx

```diff
@@ -1,37 +1,58 @@
-import React, { useState } from "react";
 import { Button, LoadingOverlay, Stack, TextInput } from "@mantine/core";
```

Comment:
Should be able to remove LoadingOverlay because it has been replaced by the LoadingOverlayAndCancel component. 
```suggestion
import { Button, Stack, TextInput } from "@mantine/core";
```